vnext/Microsoft.ReactNative.Cxx/JSValue.cpp

```diff
@@ -467,6 +471,11 @@ JSValue::~JSValue() noexcept {
     case JSValueType::String:
       m_string.~basic_string();
       break;
+    case JSValueType::Boolean:
+    case JSValueType::Int64:
+    case JSValueType::Double:
+    case JSValueType::Null:
```

Comment:
Should it be just 'default:'?

Reply:
@vmoroz  - I debated myself on this :) I think it's good to be explicit so that when/if a new enum value is added, it doesn't get default behavior without us knowing about it, what do you think?